README.md

```diff
@@ -20,6 +20,11 @@ It leverages [Clippy lints](https://rust-lang.github.io/rust-clippy/master/) to
 * Import it into SonarQube
    
 set analysis parameter `community.rust.clippy.reportPaths=<CLIPPY REPORT FILE>`
+
+* Optionally import tests measures (`junit` report)
+
+use `sonar.test.reportPath`
```

Comment:
I would change `sonar.test.reportPath` into `sonar.rust.test.reportPath` to make it plugin specific

Reply:
Change made in 22e6af84a8ac085ecbaa9648e71c83dc9ce0a10a